charts/newrelic-infrastructure-v3/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: newrelic-infrastructure-v3
 description: A Helm chart to deploy the New Relic Kubernetes monitoring solution
-version: 3.0.8
+version: 3.0.9
```

Comment:
You are right again, I bumped the infrastructure chart by mistake thinking I had to bump the infra-v3 chart. But now I don't even have to bump the v3 one because it was already there and I duplicated it.